livewordcloud.py

```diff
@@ -25,12 +25,17 @@ def gen_wordcloud(field="title"):
     resp = requests.get("https://publications-covid19.scilifelab.se/publications.json")
     txt = resp.json()
 
-    # the below level of normalisation will give access to abstract and the title -
+    # normalisation at this level accesses title/abstract
     # authors requires further 'digging' in the .json
     df = pd.json_normalize(txt["publications"])
+    df.replace("antibody", "antibodies",regex=True, inplace=True)
```

Comment:
Minor space missing here: `,regex=True,`

Reply:
I added it :) thanks